ocsf-import/package.yaml

```diff
@@ -0,0 +1,591 @@
+id: ocsf
+name: OCSF
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+description: |
+  A package that handles onboarding of OCSF mapping pipeline topics.
+  Based on OCSF version 1.3.0.
+  See also: https://schema.ocsf.io/1.3.0/
+
+pipelines:
+  # System Activity (1)
+  import-oscf-filesystem-activity:
+    name: "Import OCSF File System Activity"
```

Comment:
yamllint complains for me locally about the unnecessary double quotes, mind removing them? We should consider setting up a CI job here that does basic pipeline validation and runs a liner like yamllint.

Reply:
+1 for CI job.

Reply:
`yamllint` does not complain for me, locally. :grimacing: 

I removed the double quotes in https://github.com/tenzir/library/pull/10/commits/ac0f81363bafb8be0cc1f6885d215a914e840f99.